<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddStation</class>
 <widget class="QDialog" name="AddStation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>252</width>
    <height>326</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>252</width>
    <height>284</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>252</width>
    <height>8888</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lineName">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>35</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <family>微软雅黑</family>
              </font>
             </property>
             <property name="text">
              <string>站点名称：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="staName">
             <property name="minimumSize">
              <size>
               <width>140</width>
               <height>35</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>140</width>
               <height>35</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="font">
              <font>
               <family>微软雅黑</family>
              </font>
             </property>
             <property name="text">
              <string>经度：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="lng">
             <property name="minimumSize">
              <size>
               <width>140</width>
               <height>35</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>140</width>
               <height>35</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>微软雅黑</family>
              </font>
             </property>
             <property name="decimals">
              <number>6</number>
             </property>
             <property name="minimum">
              <double>115.903985000000006</double>
             </property>
             <property name="maximum">
              <double>116.904685999999998</double>
             </property>
             <property name="singleStep">
              <double>0.010000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <family>微软雅黑</family>
              </font>
             </property>
             <property name="text">
              <string>纬度：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="lat">
             <property name="minimumSize">
              <size>
               <width>140</width>
               <height>35</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>140</width>
               <height>35</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>微软雅黑</family>
              </font>
             </property>
             <property name="decimals">
              <number>6</number>
             </property>
             <property name="minimum">
              <double>39.451555999999997</double>
             </property>
             <property name="maximum">
              <double>40.353216000000003</double>
             </property>
             <property name="singleStep">
              <double>0.010000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QCheckBox" name="entrance">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>35</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>35</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
            </font>
           </property>
           <property name="text">
            <string>不可进站</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="exit">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>35</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>35</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
            </font>
           </property>
           <property name="text">
            <string>不可出站</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="addSta">
         <property name="minimumSize">
          <size>
           <width>92</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>92</width>
           <height>35</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>添加站点</string>
         </property>
         <property name="icon">
          <iconset resource="resource.qrc">
           <normaloff>:/icon/icon/add.png</normaloff>:/icon/icon/add.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="manageIntersite">
         <property name="minimumSize">
          <size>
           <width>92</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>92</width>
           <height>35</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>管理连接</string>
         </property>
         <property name="icon">
          <iconset resource="resource.qrc">
           <normaloff>:/icon/icon/settings.png</normaloff>:/icon/icon/settings.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QPushButton" name="deleteStation">
         <property name="minimumSize">
          <size>
           <width>92</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>92</width>
           <height>35</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>删除站点</string>
         </property>
         <property name="icon">
          <iconset resource="resource.qrc">
           <normaloff>:/icon/icon/delete.png</normaloff>:/icon/icon/delete.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
